### ISSUE TITLE
Feature: Add section in the docs for setting up cross section libraries

### ISSUE THREAD

**Issue by yardasol** [arfc/saltproc#79]
**Background and motivation**
For all the details, see the discussion in #76 as well as issues referenced there. 

TL;DR, we currently have no instructions or references in the docs regarding cross section library setup.

**Description of idea**
We should add to a section to the docs that give instructions -- or at the very least, provide references for instructions -- on setting up cross section libraries for each supported depletion code.

**Implementation details**
- Add a new `.rst` file or modify `installation.rst` under `docs/` with instructions or references to instructions on setting up cross section libraries for each supported depletion code.
- Updates should deploy to gh-pages automatically

**Potential snags**
Can't think of any.
